packages/backend/validation-utils/src/JoiError.ts

```diff
@@ -9,12 +9,12 @@ import * as Joi from "joi";
  *
  * @param error
  */
-export default class JoiError extends APIError<null> {
+export default class JoiError extends APIError {
```

Comment:
just to see if this works really but also could be nice for better typing:

`export default class JoiError extendsAPIError<Joi.ValidationError["details"]>`

Reply:
yup, `APIError<Record<string, Joi.ValidationError["details"]>>` works! 🙌